packages/server/src/api/epicRouter.ts

```diff
@@ -133,11 +132,17 @@ export const buildEpicRouter = (
             variant.tickerSettings && tickerData.addTickerDataToSettings(variant.tickerSettings);
         const showReminderFields = variant.showReminderFields ?? getReminderFields();
 
+        // HERE - limit to a given regional choiceCardAmounts
+        const contributionAmounts = choiceCardAmounts.get();
+        const requiredRegion = countryCodeToCountryGroupId(targeting.countryCode ?? 'GB');
+        const mvtId = targeting?.mvtId ?? 0;
```

Comment:
`targeting.mvtId` is also used (with a different default value!) earlier in this function - I think it's best to initialise `mvtId` once at the top.
We should really make it a mandatory field in the targeting payload, but that's for another day

Reply:
Will do

Reply:
Fixed